app.js

```diff
@@ -0,0 +1,209 @@
+console.log('hello from app.js')
+
+// select DOM elements
+const squares = document.querySelectorAll('.square')
+const startBtn = document.querySelector('.startBtn')
+const resetBtn = document.querySelector('.resetBtn')
+const p12 = document.querySelectorAll('.player')
+const p1 = document.querySelector('#p1')
+const p2 = document.querySelector('#p2')
+const pScores = document.querySelectorAll('.pScore')
+const gamesPlayed = document.querySelector('#gamesPlayed')
+
+
+const players = [
+    {
+        player: 1,
+        myTurn: false,
+        char: 'X',
+        wins: 0
+    },
+    {
+        player: 2,
+        myTurn: false,
+        char: 'O',
+        wins: 0
+    }
+]
+
+const game = {
+    squaresRemaining: 9,
+    play: false,
+    games: 0,
+    gamePlay: () => {
+        resetGame()
+        game.squaresRemaining = 9
+        game.play = true
+        firstTurn()
+        makeSelection()
+    },
+    checks: () => {
+        if(checkCat()) {
+            if(check3InRow()){
+                if(check3InCol()){
+                    if(check3Diag()){
+                        nextTurn()
+        }}}}
+    },
```

Comment:
the contents of this function can be condensed into separate checks that initiate an endGame function so you don't have to apply them in a nested format. ex. if the check3Diag function returns true, just end the game without having to have run the other check functions.